tests/cli/test_docs.py

```diff
@@ -296,3 +296,123 @@ def test_docs_list(
         my_caplog=caplog,
         click_result=result,
     )
+
+
+@pytest.fixture
+def context_with_site_built(titanic_data_context_stats_enabled_config_version_3):
+    context = titanic_data_context_stats_enabled_config_version_3
+    context.build_data_docs()
+    obs_urls = context.get_docs_sites_urls()
+    assert len(obs_urls) == 1
+    expected_index_path = os.path.join(
+        context.root_directory,
+        context.GE_UNCOMMITTED_DIR,
+        "data_docs",
+        "local_site",
+        "index.html",
+    )
+    assert os.path.isfile(expected_index_path)
+    return context
+
+
+@mock.patch(
+    "great_expectations.core.usage_statistics.usage_statistics.UsageStatisticsHandler.emit"
+)
+def test_docs_clean_no_site_specified_raises_helpful_error(
+    mock_emit, caplog, monkeypatch, context_with_site_built
+):
+    context = context_with_site_built
+    runner = CliRunner(mix_stderr=True)
+    monkeypatch.chdir(os.path.dirname(context.root_directory))
+    result = runner.invoke(
+        cli,
+        "--v3-api docs clean",
+        catch_exceptions=False,
+    )
+    stdout = result.stdout
+    assert result.exit_code == 1
+    assert (
+        "Please specify --all to remove all sites or specify a specific site using --site_name"
+        in stdout
+    )
+    assert "Cleaned data docs" not in stdout
+    assert mock_emit.call_count == 2
+    assert mock_emit.call_args_list == [
+        mock.call(
+            {"event_payload": {}, "event": "data_context.__init__", "success": True}
+        ),
+        mock.call(
+            {
+                "event": "cli.docs.clean",
+                "event_payload": {"api_version": "v3"},
+                "success": False,
+            }
+        ),
+    ]
+    expected_index_path = os.path.join(
+        context.root_directory,
+        context.GE_UNCOMMITTED_DIR,
+        "data_docs",
+        "local_site",
+        "index.html",
+    )
+    assert os.path.isfile(expected_index_path)
+
+    assert_no_logging_messages_or_tracebacks(
+        my_caplog=caplog,
+        click_result=result,
+    )
+
+
+@pytest.mark.parametrize(
+    "invocation",
+    [
+        "--v3-api docs clean --all",
+        "--v3-api docs clean -a",
+        "--v3-api docs clean --site-name local_site",
+        "--v3-api docs clean -s local_site",
+    ],
+)
+@mock.patch(
+    "great_expectations.core.usage_statistics.usage_statistics.UsageStatisticsHandler.emit"
+)
+def test_docs_clean_vali_site_name_and_all_flag_combinations_clean_sites(
```

Comment:
I just had to nitpick something 😂 
```suggestion
def test_docs_clean_valid_site_name_and_all_flag_combinations_clean_sites(
```

Reply:
nice catch!